omod/src/main/java/org/openmrs/module/providermanagement/fragment/controller/PersonNameFragmentController.java

```diff
@@ -14,13 +14,35 @@
 
 package org.openmrs.module.providermanagement.fragment.controller;
 
-import org.openmrs.layout.web.name.NameSupport;
+import java.lang.reflect.*;
+
 import org.openmrs.ui.framework.fragment.FragmentModel;
 
 public class PersonNameFragmentController {
 
-    public void controller(FragmentModel model) {
-        model.addAttribute("layoutTemplate", NameSupport.getInstance().getDefaultLayoutTemplate());
-    }
+	/**
+	 * @param model
+	 * @throws ClassNotFoundException
+	 * @throws SecurityException
+	 * @throws NoSuchMethodException
+	 * @throws InvocationTargetException
+	 * @throws IllegalArgumentException
+	 * @throws IllegalAccessException
+	 */
+
 
+	public void controller(FragmentModel model) throws ClassNotFoundException, NoSuchMethodException, SecurityException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
+		
+			Class<?> cls1 = Class.forName("org.openmrs.layout.LayoutSupport");
```

Comment:
Thanks @josephbate .
to make this backward compatible , this should only run conditionally in the higher OpenMRS version where we have this change in the class location ,or else we should directly import the class in the older versions of OpenMRS